LibraryApp/src/redux/auth/actions.js

```diff
@@ -1,11 +1,16 @@
-import { login as loginService, setCurrentUser as setCurrentUserStorage } from '@services/AuthService';
+import {
```

Comment:
You will have too many imports here, import everything from services and use `services.login` and etc